app/uk/gov/hmrc/trackingconsentfrontend/assets/test/interfaces/optimizelyCommunicatorFactory.spec.ts

```diff
@@ -2,6 +2,43 @@ import optimizelyCommunicatorFactory from '../../src/interfaces/optimizelyCommun
 
 describe('sendPreferences', () => {
   let testScope
+
+  const getScripts = () => Array.from(document.getElementsByTagName('script'))
+  const getScriptSrcs = () => getScripts().map(e => e.getAttribute('src'))
+  const exampleTCScriptURL = '/tracking-consent/tracking.js'
+
+  const setupScriptWithID = (id, scriptSrc: string) => {
```

Comment:
I think this setup code can be simplified. We can just do something like:
```typescript
  const setupScriptWithID = (id, scriptSrc: string) => {
    document.getElementsByTagName('html')[0].innerHTML = ''
    const head = document.head
    const scriptTagWithAttr = document.createElement('script')
    if (scriptSrc) {
      scriptTagWithAttr.setAttribute('src', scriptSrc)
    }
    if (id) {
      scriptTagWithAttr.setAttribute('data-optimizely-id', id)
    }
    head.appendChild(scriptTagWithAttr)
  }
```
The first line resets the document so we don't need the cleanup.